earth_enterprise/rpms/build.gradle

```diff
@@ -41,6 +41,40 @@ buildscript {
 
 apply plugin: "nebula.ospackage"
 
+def osId = null
+def osVersionId = null
+
+if (new File('/etc/os-release').exists()) {
```

Comment:
Doesn't this conflict with pr/606 which relocated this to a Platform class?

Reply:
The other pull request needs to be merged into this one.  Or, if the other is merged into `master`, `master` would be merged into this one.  This one is currently behind.

Reply:
Indeed, the other one has been merged into master, and now breaks this one...